app/controllers/application_controller.rb

```diff
@@ -90,7 +90,7 @@ def check_accounts_enabled
 
   helper_method :accounts_enabled?
   def accounts_enabled?
-    Rails.configuration.feature_flag_govuk_accounts
+    Rails.configuration.feature_flag_govuk_accounts && RestClient.get(Services.accounts_api).code != 503
```

Comment:
Does this need some exception handling?

https://rubydoc.info/gems/rest-client/

> - for result codes between 200 and 207, a RestClient::Response will be returned
> - for result codes 301, 302 or 307, the redirection will be followed if the request is a GET or a HEAD
> - for result code 303, the redirection will be followed and the request transformed into a GET
> - for other cases, a RestClient::ExceptionWithResponse holding the Response will be raised; a specific exception class will be thrown for known error codes